filtering-stories-by-status-controller

From: Bryan Larsen <bryan@larsen.st>

If you try to use the filter widget, you'll see it adds a `status` parameter in the query string. We need to pick that up and do something useful with it in the Projects controller at `app/controllers/projects_controller.rb`.  To do so, we will use an [association scope](/manual/scopes#association_scopes).

SHOW_PATCH

Status filtering should now be working.
---
 app/controllers/projects_controller.rb |    1 +
 1 file changed, 1 insertion(+)

diff --git a/app/controllers/projects_controller.rb b/app/controllers/projects_controller.rb
index 3c3b39b..9a6edbf 100644
--- a/app/controllers/projects_controller.rb
+++ b/app/controllers/projects_controller.rb
@@ -9,6 +9,7 @@ class ProjectsController < ApplicationController
     @stories = @project.stories.
       search(params[:search], :title).
       order_by(parse_sort_param(:title, :status))
+    @stories = @stories.status_is(params[:status]) if params[:status] && !params[:status].blank?
     hobo_show
   end
 
